Add Color Utilities

techpixel opened the pull request:
Create a utility that returns ANSI color codes from either a Hexdecimal, RGB, or 8-bit color palette. This would be useful (compared to using a third party color library) because it is supported by repl.it and allows a much larger selection of colors.

In short - Provide users a easy way to get a large selection of colors that are supported by the repl.it terminal.

Scoder12 commented:
Also, I almost forgot: Please run `poetry install` and then `flake8` and `black`. These lint and format the source code respectively. This will make indentation consistent and require docstrings and type annotations.

techpixel commented:
I've added in the changes you requested. I renamed `rgb` to `color`, and made `hexdec` a classmethod. I moved/renamed utils to termutils and changed __init__ accordingly.

I also ran flake8 and black as you asked.

techpixel commented:
Done!
